packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -1,331 +1,472 @@
 <template>
-  <div
-    class="signup"
+  <section 
+    class="signup" 
     vue-if="!isRegionLoading"
   >
-    <div class="signup--content">
-      <div class="signup--form">
-        <div v-if="isRegionLocked">
-          <p>We believe that you are located in a region recently impacted by the US sanctions and we are no longer able to process this request. This policy is in effect until further notice.</p>
-        </div>
-        <div v-else>
-          <form
-            id="signupForm"
-            @submit="submitForm"
-          >
-            <div class="row">
-              <label
-                class="field-wrapper"
-                for="email"
+    <div class="signup__wrapper">
+      <!-- <div v-if="isRegionLocked">
+        <p>We believe that you are located in a region recently impacted by the US sanctions and we are no longer able to process this request. This policy is in effect until further notice.</p>
+      </div> -->
```

Comment:
Are we removing this?

Reply:
no, this code is needed, uncommented the code